code/client/src/components/Pages/Reset_token.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './Reset_token.css'
+import ReactDOM from 'react-dom'; //added to retrive token from url
+import {BrowserRouter as Router, Switch, Route, Link, useParams } from "react-router-dom";
+
+function Reset_token() {
+
+      let { token } = useParams();
+
+      return (
+
+        <div className="reset_token">
+
+          <h1>Reset Password</h1>
+
+          <form action={"api/user/reset/"+token} method="POST">
```

Comment:
Should the token be a hidden input in the form instead of in the api request url? I remember talking about this at our meeting on Thursday and wanted to check if what you have here works better or something like that.

Reply:
I'm using this "let { token } = useParams();" method to retrieve the token from the url. There might be a better way to do this but I don't have a solution for now. So the token has to be appended to the url for things to work the way I've set them up.